app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -531,37 +572,37 @@ private void checkForRateDialog() {
     visitCounterPref.setCount(tempVisitCount);
 
     if (tempVisitCount >= 5
-        && !visitCounterPref.getNoThanksState()
-        && NetworkUtils.isNetworkAvailable(this) && !BuildConfig.DEBUG) {
+            && !visitCounterPref.getNoThanksState()
```

Comment:
4 space line continuations. Don't edit unrelated code please.

Reply:
@mhutti1 Oh I'm sorry, I guess my android studio automatically changed it. I'll change it.